lib/phoenix_live_view/test/live_view_test.ex

```diff
@@ -436,6 +443,17 @@ defmodule Phoenix.LiveViewTest do
     diff |> Diff.to_iodata() |> IO.iodata_to_binary()
   end
 
+  @doc """
+  Turns the output of a functional component into a HTML string that can be
+  introspected.
+
+  ## Examples
+
+      assert render_heex(MyComponent.display_user(%{user: %User{name: "Chris"}})) =~ "<h3>Chris</h3>"
+
+  """
+  def render_heex(rendered_struct), do: rendered_struct |> Phoenix.HTML.Safe.to_iodata() |> IO.iodata_to_binary()
```

Comment:
Should we allow an anonymous function to be given to `render_component` instead?

Reply:
I'd be fine with that also, I just want a way to test functional components in the same way as we did before for stateless ones, I wonder if it's less confusing to have a new function (even if with a different name) given the new separation of components?

Reply:
I see. I am wondering, what if we add Phoenix.HTML.html_to_string then? Pretty much the same steps here.

Reply:
👍

Reply:
I'd like to update the docs here (and indeed would be happy to alter this PR to this affect) mentioning the way to render these new function components, so people updating (such as my past self :joy:) can easily find it :)